testdata/baselines/reference/submodule/compiler/constEnumErrors.js.diff

```diff
@@ -1,45 +1,10 @@
 --- old.constEnumErrors.js
 +++ new.constEnumErrors.js
-@@= skipped -48, +48 lines =@@
+@@= skipped -46, +46 lines =@@
+ }
+
  //// [constEnumErrors.js]
- var E;
+-var E;
```

Comment:
This looks like it might have regressed; now there's no E and the code will crash at runtime.